packages/app/src/webui/preload.js

```diff
@@ -10,11 +10,16 @@ if (parse(window.location.href).protocol === 'station:') {
   const handler = handlers.find(h => h.hostname === hostname);
 
   if (handler && handler.hostname === 'multi-instance-configurator') {
-    const services = require('../services/servicesManager').default;
+    // const services = require('../services/servicesManager').default;
```

Comment:
What's the issue here?

Reply:
We can't change `bx` object. It can be defined only once.
Anyways we can't use previous approach because `services` object does not exist in isolated context

Reply:
Indeed, this could be tricky to migrate to isolated context.
As we only use the `manifest` service here, we should probably create a proper interface usable through contextBridge then.
Do you know what is breaking without `window.bx.manifest` currently?

Reply:
Here is the place we use it.
https://github.com/getstation/desktop-app/blob/50befc364ff13992f9b6b23712f991f7db2bd9b8/packages/app/src/applications/multi-instance-configuration/webui/MultiInstanceConfigurator.tsx#L103

But have you noticed that I moved the implementation to  `packages/app/src/plugins/webview-preload.js`?

Reply:
> But have you noticed that I moved the implementation to `packages/app/src/plugins/webview-preload.js`?

Oh I missed that. Nice work! So it looks good to me now :)